ide/docker/bin/xcsoar-compile

```diff
@@ -19,13 +19,16 @@ case "${1}" in
       "UNIX")
         make TARGET="${1}" -j"${CPUS}" all
       ;;
+      "UNIX-SDL")
+        make TARGET=UNIX OPENGL=n ENABLE_SDL=y USE_SDL2=y -j"${CPUS}" all
+      ;;
       "PC")
         make TARGET="${1}" -j"${CPUS}" all
       ;;
       "WIN64")
         make TARGET="${1}" -j"${CPUS}" all
       ;;
-      "*")
+      *)
```

Comment:
How is this change related to running XCSoar inside docker?

Reply:
True, this is unrelated, however, it is still a bug in the current file. We can remove it here if you wish so.

Reply:
+1 to removing it. Changes that are not related to this PR should not be in this PR. But feel free to open another PR for it if it makes sense to do so!

Reply:
Please list the target UNIX-SDL also in the valid target lists on line 29

Reply:
done